pkg/crc/machine/machine.go

```diff
@@ -401,3 +414,19 @@ func unsetMachineLogging() {
 	logging.CloseLogFile()
 	logging.SetupFileHook()
 }
+
+func checkClusterCertsValidity(driver drivers.Driver) error {
```

Comment:
@anjannath can we have this function away from machine side? may be create an another package call `cluster` and have all the cluster related checks there? this kind of function will explode the `machine.go` file and would be become something like we had in `minishift/start.go` side.

Reply:
@praveenkumar Done!